py-kms/pykms_Client.py

```diff
@@ -186,6 +195,21 @@ def client_update():
         raise RuntimeError(f'Client failed to find machine configuration in kms database - make sure it contains an entry for "{clt_config["mode"]}"')
 
 def client_connect():
+
+        if clt_config['discovery'] is not None:
+          loggerclt.info(f'Using Domain: {clt_config["discovery"]}')
+          r= None
```

Comment:
` `

Reply:
No comments here, do you wish to add one ?

Reply:
Nvm